Exercise_3/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css"/>
+</head>
+<body>
+<div class="wrapper">
+    <header class="header">
+        <aside class="sign-up">
+            <a class="sign-up__emails" href="#">sign up for our weekly emails
+                <span class="sign-up__special-sign">&gt;&gt;</span></a>
+            <a class="sign-up__launching-date" href="#">goop Fragrance Launching Nov.15</a>
+            <a class="sign-up__access" href="#">Sign up for early access
+                <span class="sign-up__special-sign">&gt;&gt;</span></a>
+        </aside>
+        <h1 class="header__title">city guide</h1>
+        <aside class="sign-in">
+            <span class="sign-in__registration"><a href="#">sign in</a> | <a href="#">create account</a></span>
+            <input class="sign-in__input"/>
+            <span class="sign-in__shipping">FREE SHIPPING on US orders over $50
+            <a href="#" class="sign-in__shipping-details">Details</a></span>
+        </aside>
+    </header>
+
+    <nav class="menu">
+        <a class="menu__item" href="#">SHOP</a>
+        <a class="menu__item" href="#">GO</a>
+        <a class="menu__item" href="#">BE</a>
+        <a class="menu__item" href="#">DO</a>
+        <a class="menu__item" href="#">SEE</a>
+        <a class="menu__item" href="#">MAKE</a>
+        <a class="menu__item" href="#">GET</a>
+    </nav>
+
+    <nav class="submenu">
+        <a class="submenu__item" href="#">MORE</a>
+        <a class="submenu__item" href="#">City Guides</a>
+        <a class="submenu__item" href="#">Recipes</a>
+        <a class="submenu__item" href="#">Skincare by goop</a>
+        <a class="submenu__item" href="#">goop Label </a>
+    </nav>
+
+    <nav class="tab-menu">
+        <a class="tab-menu__item" href="#">Home</a>
+        /
+        <a class="tab-menu__item" href="#">Go</a>
```

Comment:
This shouldn't be a link because it is an active breadcrumb. Please make it simple text.